mappings/net/minecraft/item/Item.mapping

```diff
@@ -166,16 +173,20 @@ CLASS net/minecraft/class_124 net/minecraft/item/Item
 		ARG 1 key
 	METHOD method_457 setFuel (Z)Lnet/minecraft/class_124;
 		ARG 1 isFuel
+	METHOD method_458 setTexturePosition (I)Lnet/minecraft/class_124;
+		ARG 1 texturePosition
 	METHOD method_459 getMaxCount ()I
 	METHOD method_460 setMaxCount (I)Lnet/minecraft/class_124;
 		ARG 1 maxCount
 	METHOD method_461 onCraft (Lnet/minecraft/class_31;Lnet/minecraft/class_18;Lnet/minecraft/class_54;)V
 		ARG 1 stack
 		ARG 2 world
 		ARG 3 player
+	METHOD method_462 isFuel ()Z
```

Comment:
Should be `hasSubtypes` or something similar, because this returns true on items that use damage to have different types, such as different colors of dyes and coal or charcoal, and it's also used in `ScreenHandler` to determine if stacks can be merged on different damage values.